web/src/main/java/org/fao/geonet/kernel/mef/MEFVisitor.java

```diff
@@ -44,6 +45,17 @@
  * MEF version 1 visitor to process and load MEF files.
  */
 public class MEFVisitor implements IVisitor {
+
+	private RecordInfo recordInfo;
+	private String preferredSchema;
+
+	public MEFVisitor(){}
+
+	public MEFVisitor(String preferredSchema, RecordInfo ri){
```

Comment:
Not necessary to pass the whole record info structure here - you're only using the schema field from it, but see comments below.

Reply:
As discussed, this is because I modify the `RecordInfo` instance this is referencing.